### DISCUSSION TITLE
animate when in view

### DISCUSSION THREAD

**Discussion by mike-portman** [animate-css/animate.css#1453] [Q&A] [answered]
Hi. Is there an option to enable the animation only when the user has scrolled into the view of the element? If so is there a description somewhere about how to implement that?

**Comment by Arhell** [comment, marked as answer]
Try https://wowjs.uk/ 
https://michalsnik.github.io/aos/
https://webdesign.tutsplus.com/tutorials/animate-on-scroll-with-javascript--cms-36671
https://alvarotrigo.com/blog/css-animations-scroll/
https://cssanimation.rocks/scroll-animations/

**Comment by KodingKhurram** [comment]
Hello, 

Also try [https://github.com/KodingKhurram/animate.css-dynamic](https://github.com/KodingKhurram/animate.css-dynamic) :)

**Reply by Cyberchicken** [reply]
About that, can you contact me Khurram? vittorio dot zamparella at gmail dot com

**Reply by KodingKhurram** [reply]
Hello @Cyberchicken trying to reach you on messenger waiting for your reply, you can add me up on discord and we can have a discussion.

**Reply by Gergab00** [reply]
Looks good but I could never start it, in the documentation it looks easy but in practice it has not worked for me.
